src/BulkManager/BulkManager.php

```diff
@@ -248,7 +248,7 @@ public function getColumnAttributes($gridField, $record, $columnName)
     public function getColumnMetadata($gridField, $columnName)
     {
         if ($columnName == 'BulkSelect') {
-            return array('title' => 'Select');
+            return array('title' => _t('GRIDFIELD_BULK_MANAGER.SELECT_LABEL', 'Select'));
```

Comment:
For anyone seeing this in the future, note that I'm not requesting a change to `__CLASS__` here because this format matches the way translatable strings are defined in this module, which is a departure from other supported modules.